model-mesh/rbac/role_binding.yaml

```diff
@@ -14,11 +14,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
```

Comment:
If multiple instances of this component are installed on the same cluster in different namespaces, this CRB will "update" constantly switching back and forth to granting the SA in different namespaces these permissions.  The SA will periodically lose the permissions granted in the CR and may cause other intermittent issues such as errors where the pods don't have permission to access the objects.

You can see an example of another issue where this was resolved here:
https://github.com/opendatahub-io/odh-manifests/pull/545